Firmware/Configuration_adv.h

```diff
@@ -346,62 +346,9 @@ const unsigned int dropsegments=5; //everything with less than this number of st
 // 2nd and 3rd byte (LSB first) contains a 16bit length of a command including its preceding comments.
 #define CMDHDRSIZE 3
 
-/**
- * Advanced Pause for Filament Change
- *  - Adds the G-code M600 Filament Change to initiate a filament change.
- *  - This feature is required for the default FILAMENT_RUNOUT_SCRIPT.
- *
- * Requirements:
- *  - For Filament Change parking enable and configure NOZZLE_PARK_FEATURE.
- *  - For user interaction enable an LCD display, HOST_PROMPT_SUPPORT, or EMERGENCY_PARSER.
- *
- * Enable PARK_HEAD_ON_PAUSE to add the G-code M125 Pause and Park.
- */
-
-#define PAUSE_PARK_RETRACT_FEEDRATE         60  // (mm/s) Initial retract feedrate.
```

Comment:
`PAUSE_PARK_RETRACT_FEEDRATE` is used in `void unload_filament(float unloadLength)
{`

Reply:
It's not used anymore? I changed it to `FILAMENT_UNLOAD_FAST_RETRACT_FEEDRATE` and `FILAMENT_UNLOAD_SLOW_RETRACT_FEEDRATE`.